challenges/data/03.md

```diff
@@ -9,23 +9,60 @@
 
 ## 👾 Before we start the exercise
 
-- [ ] We are going to install and use `react-native-offline`
-- [ ] Have a look at [`react-native-offline` documentation](https://github.com/rgommezz/react-native-offline)
+- [ ] We are going to install and use `@react-native-community/netinfo`
+- [ ] Have a look at [`@react-native-community/netinfo` documentation](https://github.com/react-native-netinfo/react-native-netinfo)
 
 ## 👨‍🚀 Exercise 3
 
-![Offline Example](https://raw.githubusercontent.com/reactgraphqlacademy/twitter-clone-native/master/src/exercice/05/offline-example.gif)
-
 - [ ] Install the library
 
 ```console
-npm install react-native-offline
+yarn add @react-native-community/netinfo
+```
+```console
+npm install --save @react-native-community/netinfo
+```
+
+- [ ] Create a new context `Network.tsx`
+
+You should check if the user is connected to the internet and display a message when the user is not connected.
+
+```javascript
```

Comment:
Instead of this:

- [ ] Create a component/exercice/Network.tsx file

That say it's easy to maintain if we change something 

Add a task

- [ ] Copy/paste this file